src/nbl/video/device_features.json

```diff
@@ -0,0 +1,2883 @@
+{
+    "headerGuard": "_NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_",
+    "includePath": "nbl/video/",
+    "includes": [
+        "ECommonEnums.h"
+    ],
+    "namespace": "nbl::video",
+    "structName": "SPhysicalDeviceFeatures",
+    "structComment": [
+        "Usage of feature API",
+        "## LogicalDevice creation enabled features shouldn't necessarily equal the ones it reports as enabled (superset)",
+        "**RARE: Creating a physical device with all advertised features/extensions:**",
+        "auto features = physicalDevice->getFeatures();",
+        "",
+        "ILogicalDevice::SCreationParams params = {};",
+        "params.queueParamsCount = ; // set queue stuff",
+        "params.queueParams = ; // set queue stuff",
+        "params.enabledFeatures = features;",
+        "auto device = physicalDevice->createLogicalDevice(params);",
+        "**FREQUENT: Choosing a physical device with the features**",
+        "IPhysicalDevice::SRequiredProperties props = {}; // default initializes to apiVersion=1.1, deviceType = ET_UNKNOWN, pipelineCacheUUID = '\\0', device UUID=`\\0`, driverUUID=`\\0`, deviceLUID=`\\0`, deviceNodeMask= ~0u, driverID=UNKNOWN",
+        "// example of particular config",
+        "props.apiVersion = 1.2;",
+        "props.deviceTypeMask = ~IPhysicalDevice::ET_CPU;",
+        "props.driverIDMask = ~(EDI_AMD_PROPRIETARY|EDI_INTEL_PROPRIETARY_WINDOWS); // would be goot to turn the enum into a mask",
+        "props.conformanceVersion = 1.2;",
+        "",
+        "SDeviceFeatures requiredFeatures = {};",
+        "requiredFeatures.rayQuery = true;",
+        "",
+        "SDeviceLimits minimumLimits = {}; // would default initialize to worst possible values (small values for maximum sizes, large values for alignments, etc.)"
+    ],
+    "content": {
```

Comment:
don't have this stuff in the json and python...


Nobody is stopping you from making a `SPhysicalDeviceFeatures.h` that just looks like
```cpp
#ifndef __NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_
#define __NBL_VIDEO_S_PHYSICAL_DEVICE_FEATURES_H_INCLUDED_

... all the includes ...

// the big comment

struct SPhysicalDeviceLimits.h
{
#include "generated/SPhysicalDeviceLimits_members.h" // or some other appropriate path as per @AnastaZIuk 
};

#endif
```

it will be much saner

Reply:
Done